test/evergreen.yml

```diff
@@ -5957,32 +5850,6 @@ buildvariants:
     - name: unit-test-live-restore-heavy-testing
     - name: format-live-restore-heavy-testing
 
-- name: ubuntu2004-release-arm64
-  display_name: "~ Ubuntu 20.04 (ARM64, Release Build)"
-  run_on:
-  - ubuntu2004-arm64-small
-  batchtime: 720 # 12 hours
-  expansions:
-    ENABLE_TCMALLOC: 0
```

Comment:
Noticed a minor inconsistency: the corresponding v9 build variant has TC_MALLOC enabled.